src/components/common/layout/navigation.njk

```diff
@@ -8,18 +8,21 @@
     active: true if (active == "security") else false,
     href: accountSecurity,
     text: 'general.header.accountNavSecurityLink' | translate
-  },{
-    active: true if (active == "sign-out") else false,
-    href: accountSignOut,
-    text: 'general.header.signOutLink' | translate
   }]
 %}
 <nav aria-label="Account navigation" class="account-navigation">
-  <div class="govuk-width-container">
-    <ul class="account-navigation__list">
-      {% for item in navigationItems %}
-        <li class="account-navigation__list-item {{ 'account-navigation__list-item--active' if item.active else '' }}"><a class="account-navigation__link" href="{{item.href}}">{{item.text}}</a></li>
-      {% endfor %}
-    </ul>
-  </div>
+    <div class="govuk-width-container">
+        <ul class="account-navigation__list">
+            {% for item in navigationItems %}
+                <li class="account-navigation__list-item {{ 'account-navigation__list-item--active' if item.active else '' }}">
+                    <a class="account-navigation__link" href="{{ item.href }}">{{ item.text }}</a></li>
+            {% endfor %}
+            <li class="account-navigation__list-item">
+                <form action="{{ accountSignOut }}" method="post">
+                    <input type="hidden" name="_csrf" value="{{csrfToken}}"/>
+                    <button class="govuk-link account-navigation__link {{ 'account-navigation__list-item--active' if item.active else '' }} account-navigation__button" type="submit">{{ 'general.header.signOutLink' | translate }}</button>
```

Comment:
A couple questions:
What is the role of `govuk-link`?
The active modifier isn't working. That being said, is it even needed? Seeing as clicking Sign out signs out out automatically and redirects you

Reply:
Since the element is a button, the role is implicitly button. govuk-link is purely to style the button as link.
I have also removed the active modifier.

Reply:
Yes, I suppose what I mean is more... is `govuk-link` actually needed to style this item the same as the rest of the items in the navigation, or is the `account-navigation__link` class sufficient?

Reply:
You're right, it isn't needed.